mbuild/formats/par_writer.py

```diff
@@ -0,0 +1,85 @@
+import mbuild as mb
+import parmed as pmd
+from foyer import Forcefield
+
+__all__ = ['write_par']
+
+def write_par(structure, filename):
+    """ Write CHARMM Par file given a parametrized structureture """
+
+    # ATOMS
+    with open(filename, 'w') as f:
+        f.write("ATOMS\n")
+        for atom in structure.atoms:
+            f.write("MASS -1 {:8s} {:8.4f}\n".format(atom.atom_type.name, atom.mass))
```

Comment:
Does `.par`not include typed information? Just name & mass?

Reply:
There's a nonbond section, woops.